### DISCUSSION TITLE
login issue regarding open-webui

### DISCUSSION THREAD

**Discussion by jameelAB** [open-webui/open-webui#2762] [Q&A]
I keep getting this login issue after creating a user name and email/ password :
Account Activation Pending
Contact Admin for WebUI Access
Your account status is currently pending activation. To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel.

/// I'm I doing something wrong cant get access?

**Comment by JuanjoJG-WATA** [comment]
You probably registered before with another user and now that one is the admin. I would recommend you to follow this steps to reset the password on the admin account, or to install from 0 again. [Reset Admin Password](https://docs.openwebui.com/troubleshooting/#reset-admin-password)
